### DISCUSSION TITLE
[BUG][SP-API] Authorization of access to PII information to get shipping address information

### DISCUSSION THREAD

**Discussion by andrenpt** [amzn/selling-partner-api-models#2823] [General]
## I'm submitting a...

<pre><code>[ ] Regression (a behavior that used to work and stopped working in a new release)
[x ] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

Not able to get PII for the shipping information from the API

**Comment by busedemirci** [comment]
To get PII, you need to have the necessary approval from Amazon, did you get the approval?

**Reply by andrenpt** [reply]
@busedemirci  I have already raised several CASES with Amazon but they have not approved any one,
This is the first timeI have a problem like this

**Reply by sungolivia** [reply]
@andrenpt  are you saying that you have requested to get access to PII but got denied? If so you would not be able to access the PII information until you do unfortunately.

**Reply by andrenpt** [reply]
@sungolivia I understand that but their reply it's always the same and not helpfull at all our case (8350735422) needs @amazon-auto  attention. We comply with what they ask!!!
